test/integration/image-component/default/pages/invalid-weight-or-height.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
Also the file name has the same typo (should be `width`)

Reply:
Yeah. there was many mistake on me 😅 Thank you! i will rename it.

Reply:
No worries, we are always happy to get PRs 👍